TypeError when no arguments supplied

Consider this watcher_getter:

```
 proc = watcher_getter(
        name='/home/vagrant/.virtualenvs/gryphon/services/dragoman/bin/dragoman',
        kwargs=dict(
            env=dict(
                PORT=str(port),
                STORAGE_URI=dragoman_db_uri,
            ),
        ),
        checker=functools.partial(airportlocker_listening, port),
    )
```

Causes this TypeError:

```
name = '/home/vagrant/.virtualenvs/gryphon/services/dragoman/bin/dragoman', arguments = None
kwargs = {'env': {'PORT': '9000', 'STORAGE_URI': "mongodb://['localhost:47667']/translations"}}
timeout = 20, checker = <functools.partial object at 0x7fef84e1e730>

    def watcher_getter_function(name, arguments=None, kwargs=None, timeout=20, checker=None):
        executable = find_executable(name)
        assert executable, 'You have to install {0} executable.'.format(name)
    
>       cmd = [name] + arguments or []
E       TypeError: can only concatenate list (not "NoneType") to list

/home/vagrant/.virtualenvs/gryphon/local/lib/python2.7/site-packages/pytest_services/service.py:63: TypeError
```

The issue is with the bind order of that expression. A distilled example is this:

```
>>> [] + None or []
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: can only concatenate list (not "NoneType") to list
```

Probably what was intended was this:

```
>>> [] + (None or [])
[]
```

One can workaround the problem by supplying `arguments=[]`, but it would be better if services handled its own defaults.

--- a/pytest_services/service.py
+++ b/pytest_services/service.py
@@ -60,7 +60,7 @@ def watcher_getter(request, services_log):
         executable = find_executable(name)
         assert executable, 'You have to install {0} executable.'.format(name)
 
-        cmd = [name] + arguments or []
+        cmd = [name] + (arguments or [])
 
         services_log.debug('Starting {0}: {1}'.format(name, arguments))
 

